generators/app/templates/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I wouldn't rewrite the entire file. If this file changes in a future version of the base included by `react-native init` we won't contemplate those changes.
Instead, we should do something like this: https://github.com/Wolox/wolmo-bootstrap-react-native/pull/36/files#diff-599d41b6089ac48f3a66455072df2f4e